docs/demo_guide/README.md

```diff
@@ -3,7 +3,7 @@
 Apollo provides a method to run simulation if you do not have the required
 hardware.
 
-Set up the docker release environment by following the instructions in the
+First Fork and then Clone Apollo's GitHub code and then Set up the docker release environment by following the instructions in the
```

Comment:
Fork, clone and set up should be lower case letters.

Reply:
done.